vm/src/obj/objstr.rs

```diff
@@ -106,8 +106,11 @@ impl PyStringRef {
         }
     }
 
-    fn str(self, _vm: &mut VirtualMachine) -> PyStringRef {
-        self
+    fn str(object: OptionalArg<PyStringRef>, vm: &mut VirtualMachine) -> PyResult<PyStringRef> {
```

Comment:
How does this work? It's a `str.__str__` implementation, but in CPython:
```
>>> str.__str__()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: descriptor '__str__' of 'str' object needs an argument
```

Reply:
Uh... you're right, I think. I managed to confuse myself somehow. I don't need this at all.